ui/main/src/app/modules/card/services/handlebars.service.ts

```diff
@@ -238,6 +239,12 @@ export class HandlebarsService {
         });
     }
 
+    private static registerPadStart() {
+        Handlebars.registerHelper('padStart', function (nbCharactersToAdd, characterForPadding, stringToPad) {
+            return String(stringToPad).padStart(nbCharactersToAdd, characterForPadding);
```

Comment:
Maybe it would be better to rename the 'nbCharactersToAdd' param to someting like 'targetLength'.

Reply:
Yes you're totally right :+1: